install.sh

```diff
@@ -3,6 +3,9 @@
 # exit on fail
 set -e
 
+# load system info
+source /etc/lsb-release
```

Comment:
Do we know if `lsb-release` is available on all Chrom(e/ium)OS-like environments?  I haven't checked Cloudready recently but there are others like Gallium that might have a different file or location.  Maybe we should check if it exists first?

Reply:
Now it will check existent of crosh command if lsb-release is not found